test/downloader.js

```diff
@@ -69,7 +69,7 @@ exports.tests = {
 
         test.expect(1);
 
-        testFile("http://www.linuxfoundation.org", function(size) {
+        testFile("https://www.linuxfoundation.org", function(size) {
```

Comment:
This is a test for the http protocol (see test name httpStream). I would recommend to use another site that is using plain http. The corresponding https test is further below. Make sure that the test site does not use redirections.